examples/hpc/dla-driver/examples/validate_conv.rs

```diff
@@ -91,27 +84,38 @@ fn validate_conv2d_tiny() -> bool {
     dla.input_data_ready(true);
 
     while !dla.handle_handshake() {}
-    let output = dla.read_output_i32(output_size.0 * output_size.1 * 2);
+    let output: Vec<i32> = dla.read_output_i32(output_size.0 * output_size.1 * 2);
```

Comment:
Prefer leaving out extra type annotations especially when the method name clearly communicates this. The tooling will fill this in in your IDE.

Reply:
Extra type annotation removed